jinahub/indexers/storage/PostgreSQLStorage/__init__.py

```diff
@@ -41,6 +41,7 @@ def __init__(
     ):
         super().__init__(*args, **kwargs)
         self.default_traversal_paths = default_traversal_paths
+        self.embedding_dtype = None
```

Comment:
It shouldn't be an init arg then

Reply:
it is not an init arg right? I think I removed it

Reply:
This is what I see. I tried refreshing 
![image](https://user-images.githubusercontent.com/8330330/125950402-3e053a23-436e-4f77-a459-364307890161.png)

Reply:
oh I see, if not using `self.embedding`, what is your suggestion?

Reply:
ah no, it's alright. Was thinking that we should create it in the .add step, but actually the pattern is to declare all class vars in init :+1: